framework_tvb/tvb/interfaces/web/controllers/burst/exploration_controller.py

```diff
@@ -135,11 +136,11 @@ def get_series_array_discrete(self, datatype_group_gid, backPage, color_metric=N
             color_metric = None
         if size_metric == 'None':
             size_metric = None
-
+        datatype_group_gid = uuid.UUID(datatype_group_gid)
```

Comment:
If you would put this conversion after the `if self._is_compatible(` the .hex won't be necessary

Reply:
Okay

Reply:
Can you still change as I suggest and test to report back what error you get?

Reply:
Okay, I will make it like you said. The error is: 
2020-03-30 18:57:56,027 - WARNING - tvb.interface.web.controllers.decorators - Redirect converted to error: (['http://127.0.0.1:8080/burst/explore/pse_error?adapter_name=Discrete%20Parameter%20Space%20Exploration&message=Discrete%20PSE%20is%20incompatible%20%28most%20probably%20due%20to%20result%20size%20being%20too%20large%29.'], 303).

And then if I click on that link it takes me to a web page, with the message:
Incompatible visualizer: Discrete Parameter Space Exploration
Reason: Discrete PSE is incompatible (most probably due to result size being too large)

Reply:
It looks like either a deeper problem, or there must be another stack trace in another place. 
I see nothing about the exploration_controller in tis message.
Have you tried a debug ?

Do you understand why it looks "fishy" when reading the code ?

Reply:
Also, I can not see any recent change by Paula in the history of that file neither on master nor in this PS list of commits. Can you pls point me to a commit Diff ?

Reply:
Also, it looks strange the way the 2 methods `get_series_array_discrete` and `draw_discrete_exploration` are written in this controller now. I think doth should have no UUId, nor .hex, and we should also change line 162 from pse_discreet.py into `datatype_group = dao.get_datatype_group_by_gid(datatype_group_gid)`. Bet let us have a call tomorrow with screen share and discuss these better live

Reply:
Oh sorry, I have confused that with another commit. Okay, let's discuss it live.